ipn/ipnlocal/local.go

```diff
@@ -332,7 +331,8 @@ type LocalBackend struct {
 
 	// lastSuggestedExitNode stores the last suggested exit node ID and name.
 	// lastSuggestedExitNode updates whenever the suggestion changes.
-	lastSuggestedExitNode lastSuggestedExitNode
+	lastSuggestedExitNode              lastSuggestedExitNode
+	refreshReportForExitNodeSuggestion bool
```

Comment:
Perhaps add a comment to explain what `refreshReportForExitNodeSuggestion` is tracking?